### PR TITLE
Enforce Dashes in CLI Argument Names

### LINKED ISSUE #15: Change CLI args to use dashes, not underscores
It's better style [https://stackoverflow.com/questions/1253679/should-command-line-options-in-posix-style-operating-systems-be-underscore-style](https://stackoverflow.com/questions/1253679/should-command-line-options-in-posix-style-operating-systems-be-underscore-style)

### PR DESCRIPTION
**Pull request by Kyle1668**
Closes #15

### CODE REVIEW

File: elk/tests/utils_evaluation/test_eval_parser.py [hunk 1]
```diff
@@ -0,0 +1,25 @@
+import json
+from elk.utils_evaluation.parser import get_parser
+
+
+def test_args_no_lowercase():
```
Comment (norabelrose) [inline elk/tests/utils_evaluation/test_eval_parser.py:5]: hah, this seems a little overkill but sure why not

Comment (norabelrose) [reply elk/tests/utils_evaluation/test_eval_parser.py:5]: Wait but shouldn't this be named `test_args_no_underscores`?

File: elk/tests/utils_generation/test_gen_parser.py [hunk 2]
```diff
@@ -0,0 +1,15 @@
+from elk.utils_generation.parser import get_parser
+
+
+def test_args_no_lowercase():
```
Comment (norabelrose) [inline elk/tests/utils_generation/test_gen_parser.py:4]: Ditto here

**Review by norabelrose — CHANGES_REQUESTED** [review]
Almost good to go I think, just need to rename the tests and remove the trained CCS model from Git

File: elk/tests/utils_evaluation/test_eval_parser.py (hunk 1 above)
Comment (Kyle1668) [reply elk/tests/utils_evaluation/test_eval_parser.py:5]: > hah, this seems a little overkill but sure why not

Haha, perhaps. If the number of parameters begins to expect over time then such tests may be useful.

File: elk/tests/utils_generation/test_gen_parser.py (hunk 2 above)
Comment (Kyle1668) [reply elk/tests/utils_generation/test_gen_parser.py:4]: Ahhh, overlooked that. Fixed.

Comment (norabelrose) [reply elk/tests/utils_generation/test_gen_parser.py:4]: Great, will merge as soon as you push it

**Review by norabelrose — APPROVED** [review]
LGTM

**Pull request merged**